Dokument/Dateien/Religion.tex

```diff
@@ -30,6 +30,175 @@ \section{Götter}
+
+\subsection{\textbf{Erlin} - Gott des Ausgleichs und der Gerechtigkeit}
+Erlin lehrt die Menschen, die Bindungen zu ihren Mitmenschen ausgewogen zu halten. Sowohl Respektlosigkeit als auch übermäßige Ehrerbietung versäuern das Verhältnis 
+zu den Mitmenschen und muss daher vermieden werden. Dazu gehört sich an die Gesetze und Regln der Gesellschaft zu halten.\\
+\begin{itemize}
+	\item Aussehen 
+	\begin{itemize}
+		\item alter Mann 
+		\item hochgewachsen und hager
+		\item in makelloser Robe, aufrechte Haltung
+		\item Richthammer am Gürtel
+	\end{itemize}
+	\item Aspekte
+	\begin{itemize}
+		\item Gesetzgebung und Rechtsprechung
+		\item Handel
+		\item Weises Vorgehen \footnote{Leichte Überschneidung mit Faelans Aspekt}
+	\end{itemize}
+	\item Tugenden Erlins
+	\begin{itemize}
+		\item Disziplin und Respekt
+		\item stoisch und gelassen
+		\item streng mit sich selbst und Anderen
+	\end{itemize}
+\end{itemize}
+\textit{Weit bekannter Mythos Erlins}:\\
+<Erlin fällt eine Richtspruch über ...>
+
+\subsection{\textbf{Aina} - Göttin der Lebensfreude und Liebe}
+Aina mag für manche wie die schwächste der Götter aussehen - ihre Mythen berichten fast ausschließlich von Festen und Kunstwerken. Und dennoch ist Aina eine der beliebtesten 
+Götter im einfachen Volk. Jeder Reigen zum Mittsommerfest, jedes alkoholschwangere Fest und jeder stille (und jeder weniger stille) Moment der Zweisamkeit genießt Ainas Segen 
+und Unterstützung.\\
+\begin{itemize}
+	\item Aussehen 
+	\begin{itemize}
+		\item junge Frau in Festtagskleidung
+		\item häufig zum Lachen aufgelegt
+		\item attraktiv 
+	\end{itemize}
+	\item Aspekte
+	\begin{itemize}
+		\item erotische Liebe (als Vereinigung)
+		\item Feierlichkeiten
+		\item Kunst und Kultur
+	\end{itemize}
+	\item Tugenden Ainas
+	\begin{itemize}
+		\item Kreativität und Freigeistigkeit
+		\item zu Scherzen aufgelegt
+		\item charmant und fröhlich
+	\end{itemize}
+\end{itemize}
+\textit{Weit bekannter Mythos Ainas}:\\
+Vor Urzeiten, als die Welt noch jünger war, waren die Menschen eitel aufgrund ihrer eigenen Fertigkeiten. Glemric, Barde und Meister seiner Zunft, war einer dieser Menschen. 
+Er rühmte sich seiner Künste mit Flöte und Harfe und behauptete, dass selbst Aina selbst nicht mit ihm mithalten könne. Und so kam es, dass eines Abends, als die Nächte 
+kurz und der Mittsommer nahe war, eine junge Frau vor seiner Tür stand und ihn zu einem Wettbewerb herausforderte. Gäste wurden geladen, um die Kunstfertigkeit der Kontrahenten 
+zu bewerten und sich ihrer Kunst zu erfreuen. Doch jedes Stück, das Glemric aufspielte, vermochte sein Gast zu überbieten, bis letzlich sogar Glemric selbst einsehen musste, 
+dass er geschlagen war. Noch an diesem Abend gelobte er, seine Ehre wiederherzustellen und sich bei seinem Gast zu revanchieren. Und so geschah es, dass Glemric ein Jahr später 
+in derselben Nacht Aina zu einer Revanche herausforderte. 
+
+\subsection{\textbf{Faelan} - Gott der Schläue und des Glücks}
```

Comment:
Bei diesem Gott wäre es wahrscheinlich gut, nochmal etwas näher auf andere Aspekte von ihm einzugehen. Ich verstehe das grobe Konzept, allerdings fehlt mir noch ein wenig der "Nutzen" für die Religion. Also der offizielle. Sicherlich, die haben einen Geheimdienst, aber der ist eben geheim. Klar sind Spione als Teil des Heeres bekannt, aber das ist doch eine recht kleine Menschenmenge. Sicherlich ist dies der Gott der Diebe, aber mir fehlt noch ein wenig der Nutzen für "normale" Menschen. Alle anderen Götter haben da einen deutlichen Stand. Ich finde, dass auch dieser Gott noch einen braucht. Die anderen obigen Punkte sind am Ende Dinge, die sich die entsprechenden Bevölkerungsgruppen dann selbst noch rausgenommen haben :D

Also bisher wirkt er eher einfach nur wie ein Kind, dass gerne Streiche spielt und das ist in meinen Augen unzureichend :)

Reply:
Soweit ich es verstanden habe, ist die Religion älter als der momentane Klerus, oder?
Also ist die Existenz des Gottes nicht wirklich etwas, über das die Kirche zu entscheiden hat (und die Frage nach dem Nutzen ist damit auch mehr "Was nutzt die Verehrung des Gottes der Kirche" als "Was nutzt der Gott der Kirche")
Listige Götter sind recht häufig in verschiedenen Kulturen (siehe Hermes, Loki, ...) und stellen nicht zwingend nur einen Gott für Diebe dar. Reisende, Glücksritter und alle, die sich mehr auf ihren eigenen Kopf verlassen müssen, könnten in ihm ein Vorbild sehen. Das heißt nicht, dass er von der Kirche genauso gut behandelt wird wie die anderen Vier.

Alternativ kann man auch einen Gott der Gelehrsamkeit & Magie machen, schätze ich (a la Thot).

Reply:
öhm, jetzt erst hier gesehen... Die Antworten hast du ja schon im Discord erhalten :D

Reply:
Was ist der Sinn dieser Mythos Geschichte von Aina? basiert darauf ein jährliches Fest oder so?